### PR TITLE
Rider object reduced tag setting

### PR DESCRIPTION
**Pull request by michaeltorku**
### Summary changing the ride tag
Set the reduced tag for the ride object to be the default and position it to be aligned with line-height of the adjacent text.

### CODE REVIEW

File: frontend/src/components/Tag/tag.module.css [hunk 1]
```diff
@@ -1,7 +1,10 @@
 .reducedTag {
   height: 11px;
   width: 11px;
-  float: left;
+  display: inline-block;
+  margin-left: 6px;
+  margin-right: 5px;
```
Comment (fengyuli2002) [inline frontend/src/components/Tag/tag.module.css:6]: Is there special considerations on these values?
I see the width is already 11px.

File: frontend/src/components/TableComponents/TableComponents.tsx [hunk 2]
```diff
@@ -70,8 +70,8 @@ export const Row = ({
       if (typeof cell === 'string') {
         return <Cell key={idx} data={cell} />;
       }
-      const { data: cData, tag, smallTag } = cell;
-      return <Cell key={idx} data={cData} tag={tag} smallTag={smallTag} />;
+      const { data: cData, tag } = cell;
+      return <Cell key={idx} data={cData} tag={tag} smallTag={true} />;
```
Comment (fengyuli2002) [inline frontend/src/components/TableComponents/TableComponents.tsx:74]: Is it still possible to set tags as not reduced? Or this functionality is just abandoned?

**Review by fengyuli2002 — COMMENTED** [review]
I think this is a nice fix!
Just have one question on that reduced tag.
More specifically, is there still someway to manually set tags as not reduced?

**Comment by michaeltorku** [comment]
No. For now - in line with the design guidelines - we're withholding that functionality, but I've made a note of it and we will allow the user to set it later when we do a visual layout settings section in the future

File: frontend/src/components/TableComponents/TableComponents.tsx (hunk 2 above)
Comment (michaeltorku) [reply frontend/src/components/TableComponents/TableComponents.tsx:74]: abandoned for now, see my comment below

File: frontend/src/components/Tag/tag.module.css (hunk 1 above)
Comment (michaeltorku) [reply frontend/src/components/Tag/tag.module.css:6]: yes they look good and work well (responsively)

**Review by fengyuli2002 — APPROVED** [review]
Thanks for your clarification!

**Pull request merged**